res/enkf/ecl_config.py

```diff
@@ -111,11 +111,18 @@ def __init__(self, config_content=None, config_dict=None):
                 refcase_list.append(refcase)
 
             # END_DATE_KEY
-            end_date = CTime(
-                datetime.strptime(
-                    config_dict.get(ConfigKeys.END_DATE, "31/12/1969"), "%d/%m/%Y"
+            try:
+                end_date = CTime(
+                    datetime.strptime(
+                        config_dict.get(ConfigKeys.END_DATE, "31/12/1969"), "%d/%m/%Y"
```

Comment:
Any reason we are not here as well trying to first load using ISO-format?

Reply:
No, both orders work here. Changed.